packages/core/admin/ee/admin/pages/SettingsPage/pages/AuditLogs/ListView/index.js

```diff
@@ -4,53 +4,56 @@ import {
   SettingsPageTitle,
   DynamicTable,
   useRBAC,
-  useNotification,
   useFocusWhenNavigate,
-  useFetchClient,
   useQueryParams,
+  AnErrorOccurred,
 } from '@strapi/helper-plugin';
-import { HeaderLayout, ContentLayout } from '@strapi/design-system/Layout';
+import { HeaderLayout, ContentLayout, ActionLayout, Layout } from '@strapi/design-system/Layout';
+import { Box } from '@strapi/design-system/Box';
 import { Main } from '@strapi/design-system/Main';
```

Comment:
You can now use destructured and tree-shaken imports:

```suggestion
import { Box, HeaderLayout, ContentLayout, ActionLayout, Layout, Main } from '@strapi/design-system';
```

Reply:
@gu-stav this is great news! Could you point me to the change that made this possible?